src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests.csproj

```diff
@@ -44,5 +50,8 @@
       <Name>RemoteExecutorConsoleApp</Name>
     </ProjectReference>
   </ItemGroup>
+  <Target Condition="'$(TargetsWindows)' == 'true'" Name="BuildNetfxRemoteInvoker" AfterTargets="BeforeBuild">
+    <MSBuild Projects="RemoteInvokeNetfx\RemoteInvokeNetfx.csproj" Properties="Configuration=Debug; Platform=$(Platform); OutputPath=$(OutputPath); IntermediateOutputPath=$(IntermediateOutputPath)" />
```

Comment:
I chatted with @krwq briefly and apparently they need this because they need to build against desktop but didn't get why are we doing this instead of using a ProjectReference instead. @krwq do you mind giving more info on why do you need this and maybe @weshaggard might think of a better way (or less hacky) to accomplish.